packages/dao/scripts/deploy.js

```diff
@@ -76,19 +123,28 @@ module.exports = async (callback) => {
         { name: "payroll", contractName: "Payroll" },
         { name: "finance", contractName: "Finance" },
         { name: "token-manager", contractName: "TokenManager" },
-      ]
+      ],
+      { gasPrice: gasPrice }
     );
+
+    let daoId;
+    if (network === "mainnet") {
+      // TODO: replace below with the final name
+      daoId = "api3-test";
+    } else {
+      daoId = `api3-${Math.random().toString(36).substring(7)}`;
```

Comment:
Why different name for template and daoId? Does it matter? (Just curious)

Reply:
You mean why are the randomized strings different? No reason/doesn't matter. Template and DAO IDs are different because the DAO ID requires `.open` to be added in public chains so doing it this way was slightly more convenient.